Detectors/TPC/calibration/src/IDCCCDBHelper.cxx

```diff
@@ -214,6 +220,44 @@ TCanvas* o2::tpc::IDCCCDBHelper<DataT>::drawIDCZeroCanvas(TCanvas* outputCanvas,
   return canv;
 }
 
+template <typename DataT>
+TCanvas* o2::tpc::IDCCCDBHelper<DataT>::drawIDCZeroScale(TCanvas* outputCanvas, bool rejectOutlier) const
+{
+  TCanvas* canv = nullptr;
+
+  if (outputCanvas) {
+    canv = outputCanvas;
+  } else {
+    canv = new TCanvas("c_sides_IDC0_scale", "IDC0", 1000, 1000);
+  }
+  canv->cd();
+  double xsides[] = {0, 1};
+  double yTotIDC0[] = {mScaleIDC0Aside, mScaleIDC0Cside};
+  auto gIDC0Scale = new TGraph(2, xsides, yTotIDC0);
+  gIDC0Scale->SetName("g_IDC0ScaleFactor");
+  gIDC0Scale->SetTitle("Scaling Factor (IDC0);Sides;IDC0 Total (arb. unit)");
+  gIDC0Scale->SetMarkerColor(kBlue);
+  gIDC0Scale->SetMarkerStyle(21);
+  gIDC0Scale->SetMarkerSize(1);
+  gIDC0Scale->GetXaxis()->SetLabelColor(0);
+  gIDC0Scale->GetXaxis()->CenterTitle();
+
+  gIDC0Scale->Draw("ap");
+  // Draw labels on the x axis
+  TText* t = new TText();
+  t->SetTextSize(0.035);
+  char* labels[2] = {"A", "C"};
```

Comment:
```suggestion
  const char* labels[2] = {"A", "C"};
```

Reply:
Oh, i forgot it, thanks i have now included it